### ISSUE TITLE
Test colour palette with teach page UI & templates

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/Design#151]
Test @kristinashu's new colour palette with one of your typical use cases. 

Use this palette: https://redpen.io/aq798f74c9a3ad7266

If you feel the need to make further adjustments to the colours, you can use @flukeout's tint generator here: http://flukeout.github.io/color-generator/ but please do document what changes you made and why.

Let me or @kristinashu know if you have any questions!

**Comment by flukeout** [comment]
I set up the colors from the redpen here, check it out...
http://flukeout.github.io/master-template/style-guide/?theme=luke-tests

**Sample layout I used**
This is the Clubs Guide template type layout, which also has a lot of similarities to the Teach site layout.

![image](https://cloud.githubusercontent.com/assets/25212/13514394/49962638-e15d-11e5-8fd7-627b8da3a2d4.png)

**Most Universally Good Colors**
These colors worked well in this layout, they were not too bright nor too dark to serve as the background color for the navigation.
* Firefox Orange
* Market Orange
* Gecko Green
* Summit Teal
* Mobile Blue
* Aurora Purple
* Bikeshed Magenta

**Good but not probably can't use in this case**
* Dino Red also worked well, but I think won't be a useful color for this application due to the negative connotations of the color.
* Grey was also good, but too Film Noire to be used as a primary color I think.

**Too Dark Overall**
I felt these colors were a little too dark and heavy to act as the background color for the left-side bar.
* Developer Blue
* Nightly Blue

![image](https://cloud.githubusercontent.com/assets/25212/13514544/b97b1386-e15e-11e5-907e-16081749549a.png)

_spooooky_

**Too Light Overall**
* Yellow was too light, it was hard to read the white text against it.
* Light grey was too light too, but probably not intended for this use anyway

**The darkest shade feels too dark**
I used the darkened shade as the selected state in the navigation, and it often felt too dark there. I think a slightly darker value would work better overall. Check out the original 15% darkening, and suggested 8% darkening below...

![image](https://cloud.githubusercontent.com/assets/25212/13514543/ac461daa-e15e-11e5-9dd4-775d036c35ef.png)

**Mismatch between the relative feel of the lightest shades**
Compare the following blocks, both which use the 40% lightening of the base shade, they are very inconsistent.

![image](https://cloud.githubusercontent.com/assets/25212/13514575/f470fd3e-e15e-11e5-999c-88ce6d80d00d.png)

This leaves us with a couple of options...
* Tweak the "lightest" version for each color separately
* Standardize the relative darkness of all of our base shades so we can apply a more formulaic approach

**In summary**
A subset of these colors (listed above) work well when used as a background color for the main navigation and accents for copy. We should make it clear in our style guide that this is a suitable use of that color. Secondly, within those colors, we should standardize the relative brightness of the colors. This will make them feel more cohesive and will make it easier to generate the lighter versions by applying a standard conversion.

I'd also make the darkest tone a little brighter, or introduce a middle tone between primary and darkest.

**Comment by xmatthewx** [comment]
Nice analysis @flukeout. Seems to jive with @lovegushwa's comment that each color will need to be optically balanced. 

For Foxlight, it shouldn't matter if each shade has a unique value, as long as the naming conventions are clean and consistent.

**Comment by flukeout** [comment]
@xmatthewx Yeah, **OR** the base tones need to have more uniform brightness. Still hangin' on to that auto-generated holy-grail am I. But yeah, we'll probably need to fine tune per color. sigh.
